src/sentry/interfaces/contexts.py

```diff
@@ -28,8 +28,52 @@ def contexttype(cls):
 
 
 class ContextType:
-    indexed_fields = None
+    tag_promotion_mapping = None
+    """
+    NOTE: This method of context promotion is considered DEPRECATED! Snuba now
+          handles promotion of context to tag or column values. It is NOT
+          recommended to use this.
```

Comment:
Where does this actually happen in snuba? What I saw was promoting context values to columns, but not to tags directly

Reply:
You're right, it is not handled at all in snuba. This isn't legacy